spline-shadow/client-ui/src/app/components/lineage/lineage-details/schema-table/schema-table.component.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2017 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, OnInit, Input, ComponentFactoryResolver, ViewChild, ViewContainerRef } from '@angular/core';
+import { PropertyType } from 'src/app/types/propertyType';
+
+@Component({
+  selector: 'schema-table',
+  templateUrl: './schema-table.component.html',
+  styleUrls: ['./schema-table.component.less']
+})
+export class SchemaTableComponent implements OnInit {
+
+  @ViewChild('detailsLine', { read: ViewContainerRef })
+  private _detailsLine: ViewContainerRef;
+
+  dtOptions: DataTables.Settings = {};
+
+  detailRows = [];
+
+  @Input()
+  schema: any
```

Comment:
type for schema?

Reply:
This type is now "any" because I am waiting the types from the swagger generation.